sdk/logic/Azure.ResourceManager.Logic/src/autorest.md

```diff
@@ -221,7 +219,6 @@ directive:
       $.OpenAuthenticationAccessPolicies.properties.policies['x-ms-client-name'] = 'AccessPolicies';
       $.ResourceReference.properties.id['x-ms-format'] = 'arm-id';
       $.ResourceReference.properties.type['x-ms-format'] = 'resource-type';
-      $.IpAddress.properties.address['x-ms-format'] = 'ip-address';
```

Comment:
Not valid ip address? or null, or empty string?

Reply:
Service will return `address=20.98.144.224/27`, therefore will throw  Ip parse exception

Reply:
Got it, please rename the name to `CidrAddress`, thank you.